### PR TITLE
Cygnet, CKEditor, and misc fixes

### PR DESCRIPTION
**Pull request by chhill-redhat**
Don't merge this. We'll re-create an MR for a drupal.org Patternkit issue.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -13,7 +13,9 @@ const gulpConfig = {
   lib: [
     './node_modules/@json-editor/json-editor/dist/**',
     './node_modules/ajv/dist/ajv.min.js',
-    './node_modules/ajv/dist/ajv.min.js.map'
+    './node_modules/ajv/dist/ajv.min.js.map',
+    './node_modules/handlebars/dist/handlebars.js',
```
Comment (mariohernandez) [inline gulpfile.js:17]: should we only include the minified version of the file?

**Review by mariohernandez — APPROVED** [review]

File: gulpfile.js (hunk 1 above)
Comment (chhill-redhat) [reply gulpfile.js:17]: @mariohernandez , good question. We include both minified and non-minified versions of JSON Editor, so I included both versions for Handlebars, to follow that pattern (no pun intended).

**Comment by chhill-redhat** [comment]
Closing this PR in favor of the same changes on drupal.org: https://git.drupalcode.org/project/patternkit/-/merge_requests/7 (issue at https://www.drupal.org/project/patternkit/issues/3260411).

File: gulpfile.js (hunk 1 above)
Comment (mariohernandez) [reply gulpfile.js:17]: 👍

**Pull request closed without merging**